test/utils/wasm3_engine.cpp

```diff
@@ -38,7 +40,35 @@ class Wasm3Engine final : public WasmEngine
 
 namespace
 {
-const void* env_adler32(IM3Runtime /*runtime*/, uint64_t* stack, void* mem)
+M3ValueType translate_valtype(char input)
+{
+    if (input == 'i')
```

Comment:
This can be a `switch`. Fine to me as it is.

Reply:
It was just moving code around, see #741.